web/modules/custom/joinup_search/joinup_search.module

```diff
@@ -395,3 +396,25 @@ function joinup_search_views_pre_execute(ViewExecutable $view) {
   // Selecting the entity_created option itself won't affect the query.
   $view->getQuery()->sort('entity_created', 'DESC');
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function joinup_search_form_views_exposed_form_alter(array &$form, FormStateInterface $form_state, string $form_id) {
+  $view = $form_state->getStorage('view');
+  if ($view['view']->id() !== 'search') {
+    return;
+  }
+
+  $user_input = $form_state->getUserInput();
+  if (empty($user_input['keys'])) {
+    unset($form['sort_by']['#options']['relevance']);
```

Comment:
This removes the "Sort by relevance" entirely from the dropdown. This has an unwanted side effect, since if a user clicks on "Advanced search" with the intention to search on keywords, then the default option (and the one that makes the most sense) is not available to them.

When a user arrives on the search page, then the options should still include "Relevance" and this should still be set as the default option, so that when a user enters their keywords they will get relevant search results.

The expectation of the user is that they will get relevant results after entering their keywords. As it is implemented now, the user will need to take a bunch of additional steps: enter keywords, click search button, then when the results are shown they have to choose the "Relevance" option from the dropdown and launch the search again.

So I think the idea of this ticket is that the _results_ should be shown in last updated order when users arrive on this page, but the actual form should remain untouched.